scripts/train/README.md

```diff
@@ -54,11 +54,11 @@ To verify that the dataloader works, run a quick test on your `val_small` split
 # This will construct a `StreamingTextDataset` dataset from your `val` split,
 # pass it into a PyTorch Dataloader, and iterate over it and print samples.
 # Since we only provide a local path, no streaming/copying takes place.
-python ../data_prep/text_data.py --local_path ./my-copy-c4 --split val_small
+python ../../llmfoundry/data/text_data.py --local_path ./my-copy-c4 --split val_small
```

Comment:
Can you double check that the extra `../` is necessary?

Reply:
yeah if you're in `llm-foundry/scripts/train/` and want to access `llm-foundry/llmfoundry/` you need to go up twice.